src/components/navigation/navigation.e2e.ts

```diff
@@ -44,7 +44,7 @@ describe('navigation', () => {
     `);
 
     // ensure ml-auto class is rendered
-    const linkSecondary = await page.findAll(`${root} .navbar-collapse:last-child a`);
+    const linkSecondary = await page.findAll(`${root} .navbar-collapse:last-of-type a`);

```

Comment:
Now we have a symbol after navbar-collapse div, so, last-child is not working and we need to change it to last-of-type, which is a div.